CRM/Case/Form/Activity.php

```diff
@@ -372,7 +372,9 @@ public function postProcess($params = NULL) {
         }
       }
       else {
-        $statusMsg = ts("Selected Activity cannot be deleted.");
```

Comment:
@mattwire so I feel like using the api would be more correct than doing rollback at the form layer.

It seems to me that activity.delete api *should* also delete entity tags within that action - so that bit of code should be moved into whatever activity.delete api calls (ideally with a test) & the form could call activity.delete rather than all this - which feels brittle